src/mixins/TacticWatcher.ts

```diff
@@ -0,0 +1,49 @@
+import Component from 'vue-class-component'
+import Vue from 'vue'
+import { Tactic } from '@/store/modules/types'
+import { Socket } from 'vue-socket.io-extended'
+import { CanvasAction } from '@/store/modules/canvas'
+import { CanvasElement, CanvasElementHistory } from '@/types/Canvas'
+import { Action } from 'vuex-class'
+import { StageActions } from '@/store/modules/stage'
+import CenterCanvas from '@/tools/util/CenterCanvas'
+import HandleRenderShapes from '@/util/HandleRenderShapes'
+
+@Component({
+  name: 'StageWatch'
+})
+export default class TacticWatcher extends Vue {
+  @Action(`canvas/${CanvasAction.SET_CANVAS_ELEMENT}`) setCanvasElements!: (canvasElements: CanvasElement[]) => void
+  @Action(`canvas/${CanvasAction.SET_CANVAS_ELEMENT_HISTORY}`) setCanvasElementsHistory!: (canvasElements: CanvasElementHistory[]) => void
+  @Action(`stage/${StageActions.SET_STAGE_TACTIC}`) setStageTactic!: (tactic: Tactic) => void
+
+  @Socket('changeTactic')
+  onChangeTactic (tactic: Tactic[]) {
```

Comment:
This is where we listen for a changeTactic request. This will work weather a user creates a new tactic or loads an old one.